### DISCUSSION TITLE
how can i use YooptaEditor

### DISCUSSION THREAD

**Discussion by seonow** [yoopta-editor/Yoopta-Editor#158] [Q&A]
Thank you for your editor service.
It's a really great editor.
We are using your editor in our project.
However, we are facing an issue when saving and retrieving the content as JSON.
The content saves correctly, but when we try to use the JSON data as the value for the editor, it doesn't work because of the format.
How can we make the data retrieved from the database appear correctly in the editor?
please help me🙏

**Comment by maZahaca** [comment]
Thank you for your question, @seonow! It's great to hear you enjoy Yoopta.
Please share example details on what JSON data looks like.

Also, consider joining our [Telegram Dev Community](https://t.me/YooptaEditor), where we can respond faster and discuss it in detail.

**Comment by Darginec05** [comment]
@seonow can you send please how JSON data looks like?

**Comment by seonow** [comment]
i already solve it thank you for your attentions @Darginec05 @maZahaca
your editor is amazing
I hope your editor is completed.

**Comment by seonow** [comment]
@Darginec05 @maZahaca
When I use links or write in Korean in a code block, extra characters are added and cannot be deleted due to an error. If I continue writing after the link, the new text also becomes part of the link due to an error.



https://github.com/Darginec05/Yoopta-Editor/assets/113659139/a7e91beb-4769-47b4-8931-b6f43ce1a93f
